examples/lessons/11/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "luma.gl-examples-lesson11",
+  "description": "An es6 example for luma.gl.",
```

Comment:
Drop the `name` field, we are not going to publish this as a module, and we want these files to be simple.
We could keep the description but maybe give it better "attribution" (we are copying some other examples after all):
`description: The classic WebGL Lesson 11 ported to luma.gl`

Reply:
done.